data/secret_zkpp_parameters.json

```diff
@@ -0,0 +1 @@
+{"secret":"<REPLACE THIS WITH A LONG RANDOM STRING>"}
```

Comment:
Two issues here:

1. We are perfectly capable of generating "a long random string" for the user, so we should do this for them.
2. The `data/` folder is not supposed to have anything in it under version control. So this file needs to be deleted.
3. Have a look at [how the `chel serve` command will be implemented](https://github.com/okTurtles/chel/issues/29). This should be a key in the database, and the database can be in-memory, on-disk as a file, or on-disk as a record in the sqlite database. So again, this file doesn't belong here.

Reply:
> We are perfectly capable of generating "a long random string" for the user, so we should do this for them.

Not really, because this parameter is set during server configuration and should not be changed afterwards without good reason to do so (if it is, passwords will stop working). So, we can generate this as part of an installation step, but not at runtime.

Reply:
> So, we can generate this as part of an installation step, but not at runtime.

Yes, especially if this is such a sensitive value. We definitely need to be in charge of it fully.

So we can automatically create it during first run if it's not set, and prevent it from being overwritten from then on.

You can create a new prefix for this: `private/immutable/`